### PR TITLE
feat(frontend): warehouse creation, global styles

### PR DESCRIPTION
**Pull request by davo22**

### CODE REVIEW

File: backend/src/migrations/1538561344596-MultipleStorageTypes.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import {MigrationInterface, QueryRunner} from 'typeorm';
+
+export class MultipleStorageTypes1538561344596 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(`ALTER TABLE "warehouse" RENAME COLUMN "storageType" TO "storageTypes"`);
+        await queryRunner.query(`ALTER TYPE "warehouse_storagetype_enum" RENAME TO "warehouse_storagetypes_enum"`);
+        await queryRunner.query(`ALTER TABLE "warehouse" DROP COLUMN "storageTypes"`);
+        await queryRunner.query(`ALTER TABLE "warehouse" ADD "storageTypes" text NOT NULL`);
```
Comment (przemczan) [inline backend/src/migrations/1538561344596-MultipleStorageTypes.ts]: You could just remove `storageType` column and add `storageTypes`. What about `warehouse_storagetype_enum` type? is it needed after that?

File: backend/src/modules/files/controllers/files.controller.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ export class FilesController {
   @UseInterceptors(
     FileInterceptor('file', {
       limits: { fieldSize: config.getInt(ConfigKey.FILE_UPLOAD_MAX_SIZE) },
-      fileFilter: fileFilter([ 'image/gif', 'image/png', 'image/jpeg', 'image/bmp' ]),
+      fileFilter: fileFilter([ 'image/gif', 'image/png', 'image/jpeg' ]),
```
Comment (kduralek) [inline backend/src/modules/files/controllers/files.controller.ts]: I think file types should be also in config

File: frontend/src/app/app.component.scss [hunk 3]
```diff
@@ -29,7 +14,27 @@
...
+  &:not(.auth) {
+    margin: 0 30px 20px;
+    padding-top: 20px;
+    border-top: 1px solid $border-color;
+  }
+}
+
+.top {
+  display: flex;
+  flex-direction: row;
```
Comment (kduralek) [inline frontend/src/app/app.component.scss]: you don't need this - it's default value

File: frontend/src/app/onboarding/onboarding.component.ts [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  submit(): void {
+    Object.values(this.formGroup.controls).forEach(control => control.markAsTouched());
+
+    if (this.formGroup.invalid) {
+      return;
+    }
+
+    this.submitButton.disabled = true;
+    this.loading = true;
+    this.warehousesService.createWarehouse(this.formGroup.value)
```
Comment (kduralek) [inline frontend/src/app/onboarding/onboarding.component.ts]: you need to unsubscribe this in `ngOnDestroy`

File: frontend/src/app/shared/components/action-buttons/action-buttons.component.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  templateUrl: './action-buttons.component.html',
+  styleUrls: [ './action-buttons.component.scss' ]
+})
+export class ActionButtonsComponent {
+  readonly buttons$: Observable<ActionButton[]>;
+
+  constructor(
+    private readonly actionButtonsService: ActionButtonsService
+  ) {
+    this.buttons$ = this.actionButtonsService.getButtons();
```
Comment (kduralek) [inline frontend/src/app/shared/components/action-buttons/action-buttons.component.ts]: It's recommended to put subscriptions and other things not related to object creation in `ngOnInit`. Just saying ;-)

File: frontend/src/app/shared/enums/employees-range.enum.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+export enum EmployeesRange {
+  Zero = 'Zero',
```
Comment (kduralek) [inline frontend/src/app/shared/enums/employees-range.enum.ts]: You don't need to do string assigment - just use enum keys: 
```
enum Example {
    A,
    B
}

Example.A // return number 0
Example[Example.A] // return string 'A'
```

File: frontend/src/app/shared/enums/metric-type.enum.ts [hunk 7]
```diff
@@ -0,0 +1,4 @@
+export enum MetricType {
+  SquareFeets = 'SquareFeets',
```
Comment (kduralek) [inline frontend/src/app/shared/enums/metric-type.enum.ts]: Same as above

File: frontend/src/app/shared/enums/storage-type.enum.ts [hunk 8]
```diff
@@ -0,0 +1,7 @@
+export enum StorageType {
+  HighRackStorage = 'HighRackStorage',
```
Comment (kduralek) [inline frontend/src/app/shared/enums/storage-type.enum.ts]: Same as above

File: frontend/src/index.html [hunk 9]
```diff
@@ -8,7 +8,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/png" sizes="32x32" href="assets/img/favicon-32x32.png">
   <link rel="icon" type="image/png" sizes="16x16" href="assets/img/favicon-16x16.png">
-  <link href="https://fonts.googleapis.com/css?family=Rubik:300,400" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Rubik:400,400i,500&amp;subset=cyrillic,hebrew,latin-ext"
```
Comment (kduralek) [inline frontend/src/index.html]: remove subset - it's not necessary since we're using English

**Pull request merged**